### ISSUE TITLE
Future improvements to modaction interface

### ISSUE THREAD

**Issue by Aquanim** [ZeroK-RTS/Zero-K-Infrastructure#2729]
- For warnings replace "PENALTY:" with "WARNING:"
- Display non-penalty messages in admin-only display (in addition to sending them via Nightwatch PM for the user themselves)
- Display actioning admin in #zkadmin notification

**Comment by maackey** [comment]
I agree on most of that (and more... admin interface needs a lot of work).

But I think warnings shouldn't be admin-only. They should be publicly visible -- along with full penalties. Is there any reason to hide them / am I misunderstanding that part?

**Comment by sprunk** [comment]
I think this is about this thing on the userpage:
![image](https://user-images.githubusercontent.com/2573076/96177145-ae785a00-0f2d-11eb-9356-39caab1bc2a4.png)

Which always points to your own inbox (I sent you a non-punishment message, try going to gnurps' userpage and click the button, you'll see the "some message for maackey" rather than the "foo" gnurps received).

**Comment by sprunk** [comment]
As for warnings - i.e. penalties without any actual punitive component, but without ticking the "just message and not penalty" checkbox - they already are displayed publicly so I think it's working as desired.

**Comment by Aquanim** [comment]
An existing PR causes the actual message to be visible to admins on the user page. There hasn't been an infra stable in a while.

- I think random people shouldn't see this, only the message addressee and admins (admins already do but can't see the actual message due to it pointing to their own inbox). 

This I didn't do but it could be done.

**Comment by maackey** [comment]
Lol. @sprunk thanks for the demonstration, I was able to try for myself on gnurps to differentiate a "warning" from a "message from the moderators". That is one clunky and non-intuitive way to send messages/warnings.

>     * For warnings replace "PENALTY:" with "WARNING:"
Yes, but also the interface for issuing warnings needs to be fixed as well -- warnings should be a separate, explicit action. Not the result of selecting a specific arbitrary set of undocumented checkboxes. Similarly I feel the same about "message from a moderator" which has the exact same problem. 

>     * Display non-penalty messages in admin-only display (in addition to sending them via Nightwatch PM for the user themselves)
"message from a moderator" should be in a dedicated section for *all* offline messages IMO. They would appear the same as any offline message from any other user -- the only difference would be that other moderators would be able to see them. Maybe add a special highlight style to make sure they pop out. I haven't quite gotten to the admin page, or web/offline chat on my [fork](https://github.com/maackey/Zero-K-Infrastructure) -- just rearranged some things so far, but this is just an idea of where I'm headed with that. 

>     * Display actioning admin in #zkadmin notification
From my test on gnurps, it looks like the actions do get sent to #zkadmin channel already. Perhaps this is fixed already?

**Comment by sprunk** [comment]
>warnings should be a separate, explicit action. Not the result of selecting a specific arbitrary set of undocumented checkboxes

If they exist as a separate action then you still have (punishments with no effect checkboxes ticked) usable as a ghetto warning, with no practical difference. I also think they should still be weaved into a user's ban log, among regular punishments chronologically, to easily see (de)escalation.
